frontend/playwright.config.ts

```diff
@@ -181,7 +181,27 @@ function getStorageState(): {
     );
 
     // Merge loginState with featureFlags
-    storageState.cookies = storageState.cookies.concat(loginState.cookies);
+    // Comment this out if you want to supply auth cookie manually
+    // storageState.cookies = storageState.cookies.concat(loginState.cookies);
+
+    /**
+     * Uncomment this if you want to supply auth cookie manually
+     * Replace `value` string with your own auth cookie. NOTE:: the string
+     * typically starts with "ey" and ends with "="
+     */
+    storageState.cookies = [
+      {
+        name: "cxguser",
+        value: "ey...=",
+        domain: "api.cellxgene.dev.single-cell.czi.technology",
+        path: "/",
+        expires: 1965216850,
```

Comment:
Hi @tihuan, when we document this in the README, it might be a good idea to call out updating `expires` in addition `value`, or at least set it to a future value?

Reply:
Thanks for the callout, Mim! 😄 That epochtime is in the future (April 2032) 👍 

I'll close this PR draft and reference it in the ticket thank you!

Reply:
Andrew has a PR for this now. Thank you!

https://github.com/chanzuckerberg/single-cell-data-portal/pull/5023#issuecomment-1591848385